src/components/lib/VDropdown/VDropdown.vue

```diff
@@ -151,7 +151,7 @@
 
 <script>
 import { reactive, ref } from "@vue/reactivity";
-import { watch } from "@vue/runtime-core";
+import { onBeforeUnmount, onMounted, watch } from "@vue/runtime-core";
```

Comment:
I'm wondering if these import paths here are getting auto-completed by your editor. Vue exports its public API from its entry file, so we shouldn't need to look inside `@vue/reactivity` or `@vue/runtime-core`.

Would it be simpler to use this instead?

```suggestion
import { reactive, ref, onBeforeUnmount, onMounted, watch } from "vue";
```

Reply:
Yeah this would be simpler. Those paths were autocompleted by the editor